README.md

```diff
@@ -177,6 +183,20 @@ An `efs_file_system` block supports the following keys:
 
 * `source_path` - (Required) The path relative to the EFS file system root to be mounted on the virtual host. The value is passed to the `user_data` template as `efs_source_path`.
 
+`eip`
+-------
+
+An `eip` block supports the following keys:
+
+* `name` - The name of the elastic IP address to attach to this EC2 instance. This may be the name of an existing elastic IP already created in a prior step, 
+or a name to give to a new elastic IP address to be created at the time the 
+EC2 instance is created. Default is the name of the EC2 instance (given in 
+the main `name` parameter above).
+
+* `create` - (Required) False if the elastic IP already exists and should be looked up by name, true if the elastic IP will be created new as part of the creation of the EC2 instance.
```

Comment:
I would switch the true and false clauses, i.e.,

* `create` - (Required) Set to `true` if the elastic IP should be created and destroyed with the EC2 instance. Set to `false` to look up an existing elastic IP address by name. The latter case is useful if it's desirable to persist the EIP for a lifecycle longer than that of the EC2 instance.